ra/ra_test.go

```diff
@@ -4081,3 +4069,60 @@ func TestUpdateRegistrationKey(t *testing.T) {
 	test.AssertContains(t, err.Error(), "failed to update registration key")
 	test.AssertContains(t, err.Error(), "mocked to always error")
 }
+
+func TestCRLShard(t *testing.T) {
```

Comment:
optional: This does seem like a reasonable candidate for a table driven test with
```
type testCase struct {
  cdp []string
  wantShard int64
  wantErr error
}
```

Reply:
This is a good suggestion, but I'm going to take the easy path and merge as written.